validator/models/validation_config.go

```diff
@@ -53,6 +57,8 @@ type MetricDimension struct {
 	Value string `yaml:"value"`
 }
 
+var _ ValidateConfig = (*validatorConfig)(nil)
```

Comment:
This will validate if the `validatorConfig` has all the declared function by the [interface](https://stackoverflow.com/a/27804417)